app/components/layer-link/template.hbs

```diff
@@ -1,6 +1,6 @@
 {{yield}}
 
-{{edit-text showSettings=showSettings editMode=editMode layer=layer type="title"}}
+{{edit-text showSettings=showSettings editMode=editMode layer=layer type="title"  alignment=layer.settings.values.alignment}}
```

Comment:
This edit-text component is a bit problematic, I think its not quite necessary. We may look into refactoring it.